internal/controllers/syncvolume_e2e_test.go

```diff
@@ -29,15 +31,24 @@ var _ = Describe("SyncVolume Controller", func() {
 			By("By creating a new volume")
 			stosVol := api.RandomStorageOSVolume()
 
-			err := api.AddVolume(*stosVol)
+			objWrapper := model.StosObject{
```

Comment:
nit: `volWrapper` instead of `objWrapper` is a bit more descriptive

Reply:
Done